libs/product/src/facades/product-grid/product-grid-facade.interface.ts

```diff
@@ -0,0 +1,9 @@
+import { Observable } from 'rxjs';
+
+import { DaffProduct } from '../../models/product';
+
+export interface DaffProductGridFacadeInterface<T extends DaffProduct = DaffProduct> {
+	loading$: Observable<boolean>;
+	products$: Observable<T[]>;
+	dispatch: Function;
```

Comment:
Can this have a more explicit type than `Function`?

Reply:
done